src/providers/Auth.tsx

```diff
@@ -173,6 +193,10 @@ export const useAuth = () => {
   };
 
   const canWrite = (token?: string) => {
+    if (process.env.REACT_APP_IGNORE_EGO) {
```

Comment:
not sure if this repo has a thing like this but generally we put env config variables in a `config.ts` file and do any serialization there, it's good to have a central place to keep track of configs rather than referencing `process.env` directly everywhere

Reply:
as an example: https://github.com/icgc-argo/platform-api/blob/develop/src/config.ts

Reply:
Lmao I added one of those to this project in my last PR but neglected to use it here 🙈 Will update in a sec